src/software/simulated_tests/excessive_dribbling.py

```diff
@@ -0,0 +1,56 @@
+import software.python_bindings as tbots
+from proto.import_all_protos import *
+
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_geometry,
+    create_validation_types,
+)
+
+
+class ExcessivelyDribbling(Validation):
+
+    """Checks if any friendly robot is excessively dribbling the ball, i.e. for over 1m."""
+
+    def __init__(self):
+        self.continous_dribbling_start_point = None
+
+    def get_validation_status(self, world) -> ValidationStatus:
+        """Checks if any friendly robot is excessively dribbling the ball, i.e. for over 1m.
+
+        :param world: The world msg to validate
+        :returns: FAILING when the robot is excessively dribbling
+                  PASSING when the robot is not excessively dribbling
+        """
+        ball_position = tbots.createPoint(world.ball.current_state.global_position)
+        for robot in world.friendly_team.team_robots:
+            if not tbots.Robot(robot).isNearDribbler(ball_position, 0.01):
```

Comment:
Nit: Can we add a quick comment to clarify this? It took a bit longer to decipher the if statements than expected.

Reply:
added comments